From: Carsten Schoenert <c.schoenert@t-online.de>
Date: Sat, 30 Dec 2023 19:49:34 +0100
Subject: spelling: Fix misspelled word unkown

---
 ChangeLog | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/ChangeLog b/ChangeLog
index b586ca1..491675b 100644
--- a/ChangeLog
+++ b/ChangeLog
@@ -13794,7 +13794,7 @@ New include File:
 2010-10-09  Robert Larice
  * src/include/complex.h :
     !REVERT! remove typedef for _complex, which is used nowhere
-    this breaks MSVC, currently unkown why, FIXME
+    this breaks MSVC, currently unknown why, FIXME
 
 2010-10-09  Robert Larice
  * src/frontend/parse-bison.c ,
